glambie/util/timeseries_helpers.py

```diff
@@ -447,11 +470,14 @@ def resample_derivative_timeseries_to_monthly_grid(start_dates, end_dates, chang
         end dates of each time period
     changes : np.array or list
         changes between start and end date
+    return_type : str, optional
+        type in which the result is returned. Current options are: 'arrays' and 'dataframe', by default 'arrays'
 
     Returns
     -------
-    Union[np.ndarray,np.ndarray,np.ndarray]
-        (start_dates, end_dates, changes) resampled to the monthly grid
+    Union[np.array, np.array, np.array] or pd.DataFrame, depending on specified return_type
```

Comment:
Again, I think you mean Tuple here. Union[a, b] means "can be either type a or type b".

Reply:
Done, also changing everywhere else